README.md

```diff
@@ -1,27 +1,25 @@
-# bpmn-visualization
+<h1 align="center">BPMN Visualization</h1> <br>
+<p align="center">
+   <img title="BPMN Visualization" src="https://github.com/process-analytics/bpmn-visualization-js/blob/master/config/logo.png" width="812" height="170">
+</p>
 
-
-[![Build](https://github.com/process-analytics/bpmn-visualization-js/workflows/Build/badge.svg)](https://github.com/process-analytics/bpmn-visualization-js/actions)
 [![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-js?color=orange&include_prereleases)](https://github.com/process-analytics/bpmn-visualization-js/releases)
+[![Build](https://github.com/process-analytics/bpmn-visualization-js/workflows/Build/badge.svg)](https://github.com/process-analytics/bpmn-visualization-js/actions)
+[![Live Demo](https://img.shields.io/badge/demo-online-blueviolet.svg)][demo-live-environment]
 
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](CONTRIBUTING.md)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=flat-square)](CONTRIBUTING.md)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 [![License](https://img.shields.io/github/license/process-analytics/bpmn-visualization-js?color=blue)](LICENSE)
 
 `bpmn-visualization` is a TypeScript library to visualize process execution data on [BPMN](https://www.omg.org/spec/BPMN/2.0.2/)
-diagrams with
+diagrams with:
 - additional display options for execution data (highlight some transitions, counters, and more)
-- with interactive capacities (mouse hover, click)
-
+- interactive capacities (mouse hover, click)
 
-**Supported Browsers**: Chrome, Firefox, Safari, Edge.
 
+## 🎮 Demo
```

Comment:
about the emoji, I am sure this make things clearer: at a first glance, this looks appealing but as we have a lot of paragraphs, it overloads the page IMHO.
But this is a matter of taste I guess, I won't veto on this at all.

Reply:
I find that, since we have a lot of paragraphs, we can find a paragraph quite easily if you scroll quickly.

We can always ask for the opinion of someone outside the project to decide;)

Reply:
me perso I like the icons